ITensorGPU/test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-if VERSION ≤ v"1.8" && "@v#.#" ∉ LOAD_PATH
+if isless(VERSION, v"1.8") && "@v#.#" ∉ LOAD_PATH
```

Comment:
Maybe `<` instead of `isless`?

Reply:
I was getting a weird error thrown in `Jenkins` that it didn't have the `<` operator and to use `isless`. I think this bug was related to a different issue so I migrated from `isless` to `<`